stdlib/public/Darwin/Accelerate/vImage_Buffer.swift

```diff
@@ -0,0 +1,348 @@
+    ///
+    /// - Returns: An initialized vImage buffer.
+    ///
+    /// This function will instantiate and initialize a vImage buffer from a `CGImage` using a `CGImageFormat`
+    /// based on the provided image's properties.
+    public init?(cgImage: CGImage,
+                 error: inout Int,
+                 flags options: vImage.Options = .noFlags) {
+        
+        self.init()
+        
+        guard var format = vImage_CGImageFormat(cgImage: cgImage) else {
+            return nil
+        }
+        
+        error = vImageBuffer_InitWithCGImage(&self,
+                                             &format,
+                                             nil,
+                                             cgImage,
+                                             options.flags)
+        
+        if error != kvImageNoError {
+            return nil
+        }
+    }
+    
+    /// Initialize a vImage buffer with the contents of a Core Graphics image.
+    ///
+    /// - Parameter cgImage: A `CGImage` instance to be used as the source.
+    /// - Parameter options: The options to use when performing this operation.
+    ///
+    /// - Returns: An initialized vImage buffer.
+    ///
+    /// This function will instantiate and initialize a vImage buffer from a `CGImage` using a `CGImageFormat` based on the provided image's properties.
+    public init?(cgImage: CGImage,
+                 flags options: vImage.Options = .noFlags) {
+        
+        self.init()
+        
+        guard var format = vImage_CGImageFormat(cgImage: cgImage) else {
+            return nil
+        }
+        
+        let error = vImageBuffer_InitWithCGImage(&self,
+                                                 &format,
+                                                 nil,
+                                                 cgImage,
+                                                 options.flags)
+        
+        if error != kvImageNoError {
+            return nil
+        }
+    }
+    
+    //===----------------------------------------------------------------------===//
+    
+    /// Initialize a vImage buffer with the contents of a Core Graphics image,
+    /// using a supplied format and reporting any errors.
+    ///
+    /// - Parameter cgImage: A `CGImage` instance to be used as the source.
+    /// - Parameter format: A `vImage_CGImageFormat` that describes the source image.
+    /// - Parameter error: Overwritten with the error code if the operation failed.
+    /// - Parameter options: The options to use when performing this operation.
+    ///
+    /// - Returns: An initialized vImage buffer.
+    ///
+    /// This function will instantiate and initialize a vImage buffer from a `CGImage` using a provided `CGImageFormat`.
+    public init?(cgImage: CGImage,
+                 format: vImage_CGImageFormat,
+                 error: inout Int,
+                 flags options: vImage.Options = .noFlags) {
```

Comment:
As above, this signature should be:
`init?(_ cgImage: CGImage, format: vImage_CGImageFormat, flags: options: vImage.Options = .noFlags)`

Reply:
Fixed - made throwable.